src/components/application_manager/include/application_manager/resumption/resumption_data_processor.h

```diff
@@ -278,6 +278,13 @@ class ResumptionDataProcessor : public app_mngr::event_engine::EventObserver {
   app_mngr::ButtonSubscriptions GetButtonSubscriptionsToResume(
       app_mngr::ApplicationSharedPtr application) const;
 
+  /**
+   * @brief Determines whether request is successful
+   * judging from result code received from HMI
+   * @param result_code received from HMI
+   */
+  bool IsRequestSuccessful(const hmi_apis::Common_Result::eType result_code) const;
```

Comment:
@mked-luxoft  make the parameter smart object - request. not result code.

Reply:
@LuxoftAKutsan fixed in 786b27228e71bb13cdb92380a190df8642251d72